components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -1193,7 +1193,7 @@ public static void addAuthenticationResultToCache(String key, AuthenticationResu
         AuthenticationResultCacheKey cacheKey = new AuthenticationResultCacheKey(key);
         AuthenticationResultCacheEntry cacheEntry = new AuthenticationResultCacheEntry();
         cacheEntry.setResult(authenticationResult);
-        cacheEntry.setValidityPeriod(TimeUnit.MINUTES.toNanos(IdentityUtil.getOperationCleanUpTimeout()));
+        cacheEntry.setValidityPeriod(TimeUnit.MINUTES.toNanos(IdentityUtil.getTempDataCleanUpTimeout()));
```

Comment:
Why do we change to a different configuration here?

Reply:
We are using the same `TempDataCleanUpTimeout` to configure the expiry time of AuthenticationContextCache. Also,  TempDataCleanUpEnable config is used to enable the AuthenticationResultCache. So, adding this will enable consistency.